src/sinopiaServer.js

```diff
@@ -1,14 +1,9 @@
 import SinopiaServer from 'sinopia_server'
 
-import { loadState } from './localStorage'
 import Config from './Config'
 
 const instance = new SinopiaServer.LDPApi()
-const curJwt = loadState('jwtAuth')
 instance.apiClient.basePath = Config.sinopiaServerBase
-if (curJwt !== undefined) {
-  instance.apiClient.authentications['CognitoUser'].accessToken = curJwt.id_token
-}
```

Comment:
it turns out we don't need authN for the functions in this file that talk to the server.  we assume all our data is world readable, and the server calls in this file are read-only so far.  we can use the pattern seen in `ImportResourceTemplate` if we need to fix this to wire it up to a user object so that valid session info is available (so that an id token can get passed to the server).